NerdyNomads/Text-Savvy
# Setup MongoDB connection

### Changes:
- Set up MongoDB Atlas connection
  - You can use `localhost:5000` to access the data with the appropriate endpoints
  - or, you can log in to [MongoDB](https://cloud.mongodb.com/v2/62153c9249ed980e57121ba8#clusters) and view the database there.
 
![mongodb cluster](https://user-images.githubusercontent.com/44079021/155589496-364b2985-6657-4be3-a35a-b410aff58458.png)

### To Test:
- Get `.env` file from Faith and place it into the backend directory
- Follow README instructions in backend directory
- After starting the server, you should see a message "MongoDB database connection established successfully" in your terminal.
